certified-connectors/RecordedFutureNew/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1575 @@
+                                                    "description":"rules",
+                                                    "x-ms-visibility":"internal"
+                                                },
+                                                "criticality":{
+                                                    "type":"integer",
+                                                    "format":"int32",
+                                                    "description":"criticality",
+                                                    "x-ms-visibility":"internal"
+                                                },
+                                                "riskSummary":{
+                                                    "type":"string",
+                                                    "description":"Recorded Future Risk Rules Summary",
+                                                    "title":"riskSummary",
+                                                    "x-ms-visibility":"advanced"
+                                                }
+                                            },
+                                            "description":"risk"
+                                        },
+                                        "links":{
+                                            "$ref":"#/definitions/Links"
+                                        }
+                                    },
+                                    "description":"data"
+                                }
+                            }
+                        }
+                    }
+                },
+                "summary":"IP Enrichment",
+                "description":"IP Enrichment with Recorded Future data",
+                "operationId":"IP_E",
+                "x-ms-visibility":"important",
+                "parameters":[
+                    {
+                        "name":"ip",
+                        "in":"path",
+                        "required":true,
+                        "type":"string",
+                        "x-ms-visibility":"important",
+                        "description":"The IP address to lookup. Must be a single IP address",
+                        "x-ms-summary":"IP input",
+                        "x-ms-url-encoding":"single"
+                    },
+                    {
+                        "name":"fields",
+                        "in":"query",
+                        "required":true,
+                        "type":"string",
+                        "default":"intelCard,risk,links",
+                        "x-ms-visibility":"internal"
+                    }
+                ]
+            }
+        },
+        "/lookup/ip/{ip}/extension/{extension}":{
+            "get":{
+                "responses":{
+                    "200":{
+                        "description":"default",
+                        "schema":{
+                            
```

Comment:
@johannesh-recordedfuture Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.

Reply:
The response for the extension endpoints (e.g. /lookup/ip/{ip}/extension/{extension}) looks different depending on the selected extension. I don't believe swagger v2 supports different responses based on parameters. Do you offer any other way around this? 

We would prefer not to define one endpoint per extension since we have a lot of them and they would clutter up the connector with a relatively narrow use case.

Reply:
@mamurshe Can you please help the partner to provide response schema?

Reply:
@johannesh-recordedfuture Please find an example below:

` "responses": {
          "200": {
            "description": "default",
            "schema": {
              "type": "object",
              "properties": {
                "records": {
                  "type": "array",
                  "items": {
                    "type": "object",
                    "properties": {
                      "id": {
                        "type": "string",
                        "description": "Record ID"
                      },
                      "createdTime": {
                        "type": "string",
                        "description": "Record Created Time"
                      }
                    }
                  },
                  "description": "Records"
                }
              }
            }
          }
        },`